metrics/web/test/features/auth/test_utils/signed_in_auth_store_mock.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:metrics/features/auth/presentation/state/auth_store.dart';
+import 'package:mockito/mockito.dart';
+import 'package:rxdart/rxdart.dart';
+
+/// Mock implementation of the [AuthStore] ensures that a user is already logged in into the app.
+class SignedInAuthStoreMock extends Mock implements AuthStore {
```

Comment:
Since we are overriding methods, it seems like we should extend `Fake` instead of `Mock`. Also, maybe we can move this class to the `test/test_utils` package since it is used not only in the `auth` package?